test/test_qubitarrays.jl

```diff
@@ -1,4 +1,4 @@
-using PastaQ
+xusing PastaQ
```

Comment:
Looks like a mistake here?

Reply:
Yes. I'm away, but will fix it soon.I was tired and working with a different setup. Had a feeling something might be incorrect.